### PR TITLE
TFSID 924825: When attempting to call a deployed model from TabPy get "EvaluationPlaneHandler.py has no attribute 'log'"

### PR DESCRIPTION
**Pull request by sbabayan**
fixed bug where we cannot access model from tableau, extended integration test to cover this scenario

### CODE REVIEW

**Comment by 0golovatyi** [comment]
Also seems like new unit test is needed to increase code coverage

File: models/scripts/SentimentAnalysis.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 from tabpy_tools.client import Client
 from textblob import TextBlob
-from nltk.sentiment import SentimentIntensityAnalyzer
+import nltk
+nltk.download('vader_lexicon')
```
Comment (sbabayan) [inline models/scripts/SentimentAnalysis.py]: Why are these downloads necessary? Does pip installing nltk not install everything that's needed?

File: tests/integration/integ_test_base.py [hunk 2]
```diff
@@ -150,7 +150,7 @@ def _get_config_file_name(self) -> str:
         config_file = open(os.path.join(self.tmp_dir, 'test.conf'), 'w+')
         config_file.write(
             '[TabPy]\n'
-            f'TABPY_QUERY_OBJECT_PATH = {self.tmp_dir}/query_objects\n'
+            f'TABPY_QUERY_OBJECT_PATH = ./query_objects\n'
```
Comment (sbabayan) [inline tests/integration/integ_test_base.py]: If we create this directory in the root won't it not get cleaned up after the test run?

File: models/scripts/SentimentAnalysis.py (hunk 1 above)
Comment (0golovatyi) [reply models/scripts/SentimentAnalysis.py]: didn't work for me without that

File: tests/integration/integ_test_base.py (hunk 2 above)
Comment (0golovatyi) [reply tests/integration/integ_test_base.py]: the folder is created in the temp folder which is current folder when the test runs

File: models/scripts/SentimentAnalysis.py (hunk 1 above)
Comment (nmannheimer) [reply models/scripts/SentimentAnalysis.py]: Yes the download is necessary. nltk does not install the vader lexicon by defaul, which is required for sentiment analysis in nltk. It's essentially a text file that contains the known sentiment scores of various words.

File: tests/integration/integ_test_base.py (hunk 2 above)
Comment (sbabayan) [reply tests/integration/integ_test_base.py]: I see, so we don't actually need to specify {self.tmp_dir} for TABPY_STATE_PATH either?

Comment (0golovatyi) [reply tests/integration/integ_test_base.py]: Probably not. We change the folder either when starting TabPy with startup script or when pointing to config path.

File: tests/integration/test_deploy_and_evaluate_model.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import integ_test_base
+import subprocess
+from pathlib import Path
+
+
+class TestDeployModelSSLOffAuthOff(integ_test_base.IntegTestBase):
```
Comment (0golovatyi) [inline tests/integration/test_deploy_and_evaluate_model.py]: This class name is already used in another test file.

Comment (sbabayan) [reply tests/integration/test_deploy_and_evaluate_model.py]: thanks for catching that i forgot to update it

**Comment by 0golovatyi** [comment]
Still need to increase unit tests coverage.

**Pull request merged**